pandas/core/arrays/datetimelike.py

```diff
@@ -18,6 +18,7 @@
 import warnings
 
 import numpy as np
+import pandas
```

Comment:
We shouldn't be importing pandas at the top of a module like this - it will cause import cycles.

Reply:
```
grep --include \*.py -nre ".*import.*DatetimeArray"
pandas/core/ops/array_ops.py:535:            from pandas.core.arrays import DatetimeArray
pandas/core/dtypes/dtypes.py:858:        from pandas.core.arrays import DatetimeArray
pandas/core/dtypes/dtypes.py:944:        from pandas.core.arrays import DatetimeArray
pandas/core/dtypes/astype.py:105:            from pandas.core.arrays import DatetimeArray
```

Reply:
Thanks @rhshadrach, I have removed this